modules/storage/storage-accounts/blob-services/main.bicep

```
@@ -173,6 +176,7 @@ resource blobServices 'Microsoft.Storage/storageAccounts/blobServices@2022-09-01
     deleteRetentionPolicy: {
       enabled: deleteRetentionPolicyEnabled
       days: deleteRetentionPolicyEnabled == true ? deleteRetentionPolicyDays : null
+      allowPermanentDelete: deleteRetentionPolicyEnabled == true && deleteRetentionPolicyAllowPermanentDelete == true ? deleteRetentionPolicyAllowPermanentDelete : null
```

```suggestion
      allowPermanentDelete: deleteRetentionPolicyEnabled && deleteRetentionPolicyAllowPermanentDelete ? true : null
```

Would you think I this works too? It's a bit shorter

---

It is essentially the same yes...